nncf/torch/graph/pattern_operations.py

```diff
@@ -10,54 +10,82 @@
 # limitations under the License.
 from nncf.common.graph.patterns import GraphPattern
 from nncf.common.graph.patterns import merge_two_types_of_operations
+from nncf.torch.graph import operator_metatypes as om
 
 LINEAR_OPERATIONS = {
     GraphPattern.METATYPE_ATTR: [
-        "linear",
-        "conv1d",
-        "conv2d",
-        "conv3d",
-        "conv_transpose1d",
-        "conv_transpose2d",
-        "conv_transpose3d",
-        "deform_conv2d",
-        "addmm",
-        "bmm",
-        "matmul",
-        "mm",
-        "baddbmm",
+        # Linear
+        om.PTLinearMetatype,
+        om.PTModuleLinearMetatype,
+        # Conv1D
+        om.PTConv1dMetatype,
+        om.PTDepthwiseConv1dSubtype,
+        om.PTModuleConv1dMetatype,
+        om.PTModuleDepthwiseConv1dSubtype,
+        # Conv2D
+        om.PTConv2dMetatype,
+        om.PTDepthwiseConv2dSubtype,
+        om.PTModuleConv2dMetatype,
+        om.PTModuleDepthwiseConv2dSubtype,
+        # Conv3D
+        om.PTConv2dMetatype,
+        om.PTConv3dMetatype,
+        om.PTDepthwiseConv3dSubtype,
+        om.PTModuleConv3dMetatype,
+        om.PTModuleDepthwiseConv3dSubtype,
+        # Transposed conv
+        om.PTConvTranspose1dMetatype,
+        om.PTModuleConvTranspose1dMetatype,
+        om.PTConvTranspose2dMetatype,
+        om.PTModuleConvTranspose2dMetatype,
+        om.PTConvTranspose3dMetatype,
+        om.PTModuleConvTranspose3dMetatype,
+        # Deform conv
+        om.PTDeformConv2dMetatype,
+        om.PTModuleDeformConv2dMetatype,
+        # MatMul
+        om.PTMatMulMetatype,
+        # Addmm
+        om.PTAddmmMetatype,
     ],
     GraphPattern.LABEL_ATTR: "LINEAR",
 }
 
 BATCH_NORMALIZATION_OPERATIONS = {
-    GraphPattern.METATYPE_ATTR: ["batch_norm", "batch_norm1d", "batch_norm2d", "batch_norm3d"],
+    GraphPattern.METATYPE_ATTR: [om.PTBatchNormMetatype, om.PTModuleBatchNormMetatype],
     GraphPattern.LABEL_ATTR: "BATCH_NORMALIZATION",
 }
 
 GROUP_NORMALIZATION_OPERATIONS = {
-    GraphPattern.METATYPE_ATTR: ["group_norm"],
+    GraphPattern.METATYPE_ATTR: [om.PTGroupNormMetatype, om.PTModuleGroupNormMetatype],
     GraphPattern.LABEL_ATTR: "GROUP_NORMALIZATION",
 }
 
 LAYER_NORMALIZATION_OPERATIONS = {
-    GraphPattern.METATYPE_ATTR: ["layer_norm"],
+    GraphPattern.METATYPE_ATTR: [om.PTLayerNormMetatype, om.PTModuleLayerNormMetatype],
     GraphPattern.LABEL_ATTR: "LAYER_NORMALIZATION",
 }
 
-RELU_OPERATIONS = {GraphPattern.METATYPE_ATTR: ["relu", "relu_", "hardtanh"], GraphPattern.LABEL_ATTR: "RELU"}
+RELU_OPERATIONS = {
+    GraphPattern.METATYPE_ATTR: [
+        om.PTRELUMetatype,
+        om.PTRELU6Metatype,
```

Comment:
Adding new operation `relu6` to patterns, is it correct changes?

Reply:
That is a mistake, thanks for the catch!